safe-global/safe-wallet-web
# [Multichain] feat: redesign of multiaccounts and context menu for multiaccounts

## What this PR changes
- Redesigns how sub-items are displayed under multi-accounts.
- Adds context menu for multi accounts offering renaming and adding a new network
- Some UX fixes around the context menu

## How to test it
- Open the new Sidebar and observe multi chain accounts
- Try to rename an account

## Screenshots


## Checklist
